zds/settings/abstract_base/zds.py

```diff
@@ -110,8 +110,8 @@
             "mastodon": "https://framapiaf.org/@ZesteDeSavoir",
             "facebook": "https://www.facebook.com/ZesteDeSavoir",
             "twitter": "https://twitter.com/ZesteDeSavoir",
+            "github": 'https://github.com/zestedesavoir/zds-site',
```

Comment:
```suggestion
            "github": "https://github.com/zestedesavoir/zds-site",
```

Pour satisfaire le linter :)